kubecost-single-account-permissions.yaml

```
@@ -109,5 +109,5 @@ Resources:
           Action:
           - organizations:ListAccounts
           - organizations:ListTagsForResource
-          Resource: 
-          - "*"
+          Resource:
+          - !Sub 'arn:aws:s3:::${SpotDataFeedBucketName}*'
```

A `*` without a prefacing `/` is a wild card on the name, not the objects.
Though we may need to add a second resource for just the bucket.

```suggestion
          - !Sub 'arn:aws:s3:::${SpotDataFeedBucketName}/*'
```

---

this was the wrong location in this file for spot. Fixed